SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialLatencyTracer.h"
+
+#include "Async/Async.h"
+#include "Engine/World.h"
+#include "EngineClasses/SpatialGameInstance.h"
+#include "Interop/Connection/OutgoingMessages.h"
+#include "Utils/SchemaUtils.h"
+
+#include <sstream>
+
+DEFINE_LOG_CATEGORY(LogSpatialLatencyTracing);
+
+// Stream for piping trace lib output to UE output
+class UEStream : public std::stringbuf
+{
+	int sync()
```

Comment:
Is this intended to shadow the virtual call? Otherwise
```suggestion
	int sync() override
```